apps/docs/components/tabs/tabs.css

```diff
@@ -11,36 +16,29 @@
 }
 
 .hd-tabs__item-button {
+    display: flex;
     align-items: center;
-    background: none;
+    cursor: pointer;
+    background-color: transparent;
     border: none;
     border-radius: var(--hd-space-05);
-    color: var(--hd-color-neutral-text);
-    cursor: pointer;
-    display: flex;
-    font-family: var(--hd-header-font-family);
-    font-size: 0.875rem;
-    gap: var(--hd-space-1);
     padding-inline: var(--hd-space-2);
     padding-block: var(--hd-space-1);
     transition: background-color 0.15s ease-in, color 0.05s linear;
+    z-index: 1;
 }
 
 .hd-tabs__item-button--active,
 .hd-tabs__item-button:hover {
-    color: var(--hd-color-primary-text);
-}
-
-.hd-tabs__item::after {
-    content: '';
-    height: calc( var(--hd-border-size) * 2 );
-    inset: auto 0 0;
-    position: absolute;
-
+    color: var(--hd-color-neutral-text);
+    background-color: var(--hd-color-neutral-surface);
+    box-shadow: 0 0.0625rem 0 0 rgb(0 0 0 / 15%);
```

Comment:
0.0625rem could be border size

Reply:
no, because it doesn't go around the whole element. The aim here is to create a contrast effect at the bottom of the component.